### DISCUSSION TITLE
npm run setup errors

### DISCUSSION THREAD

**Discussion by Ashleeey1224** [MarkBind/markbind#2372] [Q&A]
<img width="1198" alt="Screenshot 2023-10-18 at 10 17 17 am" src="https://github.com/MarkBind/markbind/assets/105656080/96f1dce6-0740-4546-b308-0095cee892f7">
I met this issue, how can I resolve it? do I need to post to the issue board?

**Comment by tlylt** [comment]
Hi @Ashleeey1224, thank you for reporting. I'll need more information as I'm able to run `npm run setup` without the above error. May I know:
- what steps did you follow? What exact commands did you run?
- what's your node version?
- what's your npm version?

**Reply by Ashleeey1224** [reply]
Hi @tlylt, I have figured it out. I tried to use Homebrew to reinstall Graphviz and it works now.
